From 51c1826f4cbeb51aefb26e3017d7f654d4599148 Mon Sep 17 00:00:00 2001
From: Kevin Paul Herbert <kph@platinasystems.com>
Date: Wed, 1 Feb 2017 17:31:11 -0800
Subject: [PATCH 1015/1016] layouts: Create a new folder for layouts and add
 platina-mk1

Set up an initial layout with one section "bios" offset 8MiB
into a 16MiB flash.

Layouts are installed into $(PREFIX)/share/$(PROGRAM)/layouts,
i.e. /usr/local/share/flashrom/layouts on most systems.

Signed-off-by: Kevin Paul Herbert <kph@platinasystems.com>
---
 Makefile                | 2 ++
 layouts/platina-mk1.xml | 1 +
 2 files changed, 3 insertions(+)
 create mode 100644 layouts/platina-mk1.xml

diff --git a/Makefile b/Makefile
index 01a179d..e5cdf16 100644
--- a/Makefile
+++ b/Makefile
@@ -1354,8 +1354,10 @@ $(PROGRAM).8: $(PROGRAM).8.tmpl
 install: $(PROGRAM)$(EXEC_SUFFIX) $(PROGRAM).8
 	mkdir -p $(DESTDIR)$(PREFIX)/sbin
 	mkdir -p $(DESTDIR)$(MANDIR)/man8
+	mkdir -p $(DESTDIR)$(PREFIX)/share/$(PROGRAM)/layouts
 	$(INSTALL) -m 0755 $(PROGRAM)$(EXEC_SUFFIX) $(DESTDIR)$(PREFIX)/sbin
 	$(INSTALL) -m 0644 $(PROGRAM).8 $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) -m 0644 -t $(DESTDIR)$(PREFIX)/share/$(PROGRAM)/layouts layouts/*
 
 libinstall: libflashrom.a libflashrom.h
 	mkdir -p $(DESTDIR)$(PREFIX)/lib
diff --git a/layouts/platina-mk1.xml b/layouts/platina-mk1.xml
new file mode 100644
index 0000000..e4d453d
--- /dev/null
+++ b/layouts/platina-mk1.xml
@@ -0,0 +1 @@
+00800000:00ffffff bios
-- 
2.7.4

